.circleci/config.yml

```diff
@@ -24,6 +26,27 @@ jobs:
             docker info
             docker build -t ${CIRCLE_PROJECT_REPONAME}:build .
 
+      - run:
+          name: Install Docker Compose
```

Comment:
This works! Another option would be to switch to an image that has docker-compose installed by default. I think this is good for now.